src/sentry/static/sentry/app/__mocks__/api.jsx

```diff
@@ -91,12 +91,16 @@ class Client {
 
       // mock gets returned when we add a mock response, will represent calls to api.request
       mock(url, options);
+
+      const body =
+        typeof response.body === 'function' ? response.body(url, options) : response.body;
```

Comment:
This is so `response.body` can be a function of `options` (e.g. `options.query`).